### PR TITLE
mocks: add bugzilla.account.search mock endpoint (bug 1836484)

### PR DESCRIPTION
**Pull request by zzzeid**

### CODE REVIEW

**Review by dklawren — APPROVED** [review]
Looks sane to me. r=dkl

File: tests/mocks.py [hunk 1]
```diff
@@ -733,6 +733,27 @@ def comment(self, content, author=None):
     def conduit_ping(self):
         return "ip-123-123-123-123.us-west-2.compute.internal"
 
+    @conduit_method("bugzilla.account.search")
+    def bugzilla_account_search(self, phids=None, ids=None):
+        """Return a list of Bugzilla IDs and Phabricator phids, given phids or ids."""
+        if ids:
+            raise NotImplementedError("Searching by Bugzilla ID is not implemented.")
```
Comment (cgsheeh) [inline tests/mocks.py:740]: Not implemented in the test mock, or not implemented in the Conduit API endpoint?

**Review by cgsheeh — APPROVED** [review]
This looks good to me, but it's a bit hard to tell since this is unused code at the moment. I think it makes more sense to add this in the patch where it will be used in a test.

File: tests/mocks.py (hunk 1 above)
Comment (zzzeid) [reply tests/mocks.py:740]: > Not implemented in the test mock, or not implemented in the Conduit API endpoint?

In the test mock -- I can be more clear about that.

**Comment by zzzeid** [comment]
> This looks good to me, but it's a bit hard to tell since this is unused code at the moment. I think it makes more sense to add this in the patch where it will be used in a test.

This is used in https://github.com/mozilla-conduit/lando-api/pull/325, which I can land around the same time (as it depends on this one).

**Pull request merged**